lemkus_spider.py

```diff
@@ -0,0 +1,59 @@
+import json
+import re
+import scrapy
+
+
+class LemkusSpider(scrapy.Spider):

```

Comment:
Have you seen `scrapy.CrawlSpider`? Explore it and see if it can be of any benefit to us.